src/ui/setup/helpers.ts

```diff
@@ -11,12 +11,27 @@ declare global {
     prefs: typeof prefs;
     features: typeof features;
     dumpPrefs: () => string;
+    dumpBasicProcessing: () => void;
     local: () => void;
     prod: () => void;
     clearIndexedDB: () => void;
   }
 }
 
+function dumpBasicProcessing() {
+  const processing = window.sessionMetrics?.filter(
+    (data: any) => data.event == "RegionBasicProcessing"
```

Comment:
Triple equals for consistency? I feel like we should have a linter for that one.